README.md

```diff
@@ -1,40 +1,114 @@
+In this **simple demo project**, we **load a PDF file** in one side of the page, and in the other one **a form** where the volunteer will be able to transcribe the PDF page by typing the text in the input form. While this example is really simple, adapting the template to extract specific bits of information from the PDF will be really easy (you will only need to add more HTML input fields with instructions about what you want to extract from the PDF file). The idea is that you could be able for example to extract specific items from the documents, like captions, tabular data, authorship, institutions, etc.
 
 ![alt screenshot](http://img10.imageshack.us/img10/5364/pdftranscribe1.png)
 
 The provided script for creating the tasks is very simple: you only need to tell the script where is the PDF file hosted, the URL, and which pages you want to convert as tasks. By default, this demo explores the 14 pages of the example PDF file.
 
-Re-using the application for your project
+Re-using the project for your project
 =========================================
 
-You need to install the pybossa-client first (use a virtualenv):
+You need to install the pybossa-pbs library first. Use of a virtual environment
+is recommended:
+
+```bash
+    $ virtualenv env
+    $ source env/bin/activate
+```
 
 ```bash
     $ pip install -r requirements.txt
 ```
-Then, you can follow the next steps:
 
-*  Create an account in crowdcrafting.org 
-*  Copy under your account profile your API-KEY
-*  Run python createTasks.py -u http://crowdcrafting.org -k API-KEY -f http://domain/yourpdf -p numPages
-*  Open with your browser the Applications section and choose the PDF Transcribe app. This will open the presenter for this demo application.
+## Creating an account in a PyBossa server
+Now that you've all the requirements installed in your system, you need
+a PyBossa account:
+
+*  Create an account in your PyBossa server (use [Crowdcrafting](https://crowdcrafting.org) if you want).
+*  Copy your API-KEY (you can find it in your profile page).
+
+## Configure pybossa-pbs command line
+
+PyBossa-pbs command line tool can be configured with a config file in order to
+avoid typing the API-KEY and the server every time you want to take an action
+on your project. For this reason, we recommend you to actually create the
+config file. For creating the file, follow the next steps:
+
+```bash
+    $ cd ~
+    $ editorofyourchoice .pybossa.cfg
+```
+
+That will create a file. Now paste the following:
+
+```python
```

Comment:
this is is not python. Replace by `INI` https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1439